docs/getting-started/quickstart.rst

```diff
@@ -1,39 +1,19 @@
 .. _quickstart-setup:
 
-Quickstart Setup
-----------------
+Recommended Setup
+-----------------
 
-The best way to get started and learn Gammapy are the :ref:`tutorials`. For
-convenience we provide a pre-defined conda environment file, so you can
-get additional useful packages together with Gammapy in a virtual isolated
-environment. First install `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`__
-and then just execute the following commands in the terminal:
+We recommend using :ref:`virtual environments <virtual-envs>`, to do so
+execute the following commands in the terminal:
 
 .. substitution-code-block:: bash
 
     $ curl -O https://gammapy.org/download/install/gammapy-|release|-environment.yml
     $ conda env create -f gammapy-|release|-environment.yml
 
-.. note::
-
-    On Windows, you have to open up the conda environment file and delete the
-    lines with ``sherpa`` and ``healpy``. Those are optional dependencies that
-    currently aren't available on Windows.
-
-.. note::
```

Comment:
Are sherpa builds for M1 available by now for all OS versions? If not we should keep the comment or adapt all env files in https://github.com/gammapy/gammapy-webpage/tree/gh-pages/download/install to instal sherpa via pip (but only if there are wheels, I have not checked lately...)

Reply:
Indeed sherpa M1 build is not available with conda. I moved the sherpa to the pip channel (in this PR). However, I'm not sure about the wheels. But this was also done in the 1.2rc1 environment file and the installation succeed.